okhttp/src/main/java/okhttp3/internal/Util.java

```diff
@@ -233,7 +233,9 @@ public static ThreadFactory threadFactory(String name, boolean daemon) {
     List<String> result = new ArrayList<>();
     for (String a : first) {
       for (String b : second) {
-        if (comparator.compare(a, b) == 0) {
+        @SuppressWarnings("https") int value = comparator.compare(a, b);
```

Comment:
In these case studies, I used the labels `"FALSE POSITIVE"` and `"TRUE POSITIVE"` when suppressing warnings. Can you do the same so that simple searches give consistent results? You should also justify why you know that a warning is a false positive (in this case, there's obviously no URL here).

As a model, the style I always use is:
```
@SuppressWarnings("https") // FALSE POSITIVE: this code manifestly is generic over https, but the checker issues a warning related to generics
```